jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -172,7 +172,8 @@ private T call(URL url) throws IOException, RegistryException {
 
     } catch (IOException ex) {
       logError("I/O error for image [" + serverUrl + "/" + imageName + "]:");
-      logError("    " + ex.getMessage());
+      logError("    " + ex.getClass().getName());
+      logError("    " + (ex.getMessage() == null ? "(null exception message)" : ex.getMessage()));
```

Comment:
Perhaps you already thought about this, but should we really print `(null exception message)` or do
```
if (ex.getMessage() != null) {
  logError(ex.getMessage())
} 
```

Reply:
I think it's only more helpful to give more information in this error case. It's not a big deal in either way though.